src/test/kotlin/dev/stalla/RealLifeFeedsTest.kt

```diff
@@ -30,7 +30,7 @@ internal class RealLifeFeedsTest {
             assertThat(parsedPodcast, "parsed podcast").isNotNull()
 
             val tmpFile = File.createTempFile("stalla_test", "writer_output")
```

Comment:
Could this be improved by using the `TemporaryFileParameterResolver`?

Reply:
I'm not sure. I think the dynamic test lambdas don't take arguments, but I'll see what I can do.

Reply:
Was mostly wondering, it's ok if it's a no :)